src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
According to the checklist, you should avoid using the `*` selector for styling, as it impacts performance. Although this is a minor issue, it's a good practice to style only the elements that require it.